docs/stacks-guide-gcp.md

```diff
@@ -9,459 +9,451 @@ indent: true
-can install and use the [WordPress sample
-stack][sample-wordpress-stack], which depends on MySQL and Kubernetes!
+In this guide, we will set up a GCP provider in Crossplane so that we can
+install and use the [WordPress sample stack][sample-wordpress-stack], which
+depends on MySQL and Kubernetes!
 
 Before we begin, you will need:
 
-* Everything from the [Crossplane Stacks Guide][stacks-guide] before the
+- Everything from the [Crossplane Stacks Guide][stacks-guide] before the
   cloud provider setup
-  - A `kubectl` pointing to a Crossplane control cluster
+  - A `kubectl v1.15+` pointing to a Crossplane cluster
   - The [Crossplane CLI][crossplane-cli] installed
-* An account on [Google Cloud Platform][gcp]
+- An account on [Google Cloud Platform][gcp]
 
 At the end, we will have:
 
-* A Crossplane control cluster configured to use GCP
-* The boilerplate of a GCP-based project spun up
-* Support in the control cluster for managing MySQL and Kubernetes
-  cluster dependencies
-* A slightly better understanding of:
-  - The way cloud providers are configured in Crossplane
+- A Crossplane control cluster configured to use GCP
+- A typical GCP network configured to support secure connectivity between
+  resources
+- Support in Crossplane cluster for satisfying MySQL and Kubernetes claims
+- A slightly better understanding of:
+  - The way GCP is configured in Crossplane
   - The way dependencies for cloud-portable workloads are configured in
     Crossplane
 
-We will **not** be teaching first principles in depth. Check out the
-[Crossplane concepts document][crossplane-concepts] for that.
+We will **not** be covering the core concepts in this guide, but feel free to
+check out the [Crossplane concepts document][crossplane-concepts] for that.
 
 ## Install the GCP Stack
 
 After Crossplane has been installed, it can be extended with more
 functionality by installing a [Crossplane Stack][stack-docs]! Let's
 install the [stack for Google Cloud Platform][stack-gcp] (GCP) to add
-support for that cloud provider. We can use the [Crossplane
-CLI][crossplane-cli] for this operation. Since this is an infrastructure
-stack, we need to specify that it's cluster-scoped by passing the
-`--cluster` flag.
+support for that cloud provider.
 
-To install to a specific namespace, we can use the `generate-install`
-command and pipe it to `kubectl apply` instead, which gives us more
-control over how the stack's installation is handled. Everything is
-a Kubernetes object!
+The namespace where we install the stack, is also the one that the provider
+secret will reside. The name of this namespace is arbitrary, and we are calling
+it `crossplane-system` in this guide. Let's create it:
 
-```
-kubectl create namespace gcp
-kubectl crossplane stack generate-install --cluster 'crossplane/stack-gcp:master' stack-gcp | kubectl apply --namespace gcp -f -
+```bash
+# namespace for GCP stack and provider secret
+kubectl create namespace crossplane-system
```

Comment:
Is it possible for the reader to get this far without this namespace existing, due to it being created via the Crossplane Helm chart?

Reply:
That's a good question. I think we are emphasizing more on creating *a* namespace, than having this specific one. Do you think we should change it?

Reply:
I'd lean toward skipping it personally. I expect we're using an example namespace that already exists. Not a big deal though, happy to merge without it.